### ISSUE TITLE
what naming/path conventions for e2e testing when they hit?

### ISSUE THREAD

**Issue by ErichBSchulz** [ngbp/ngbp#260]
thanks Josh for earlier comment...

I'm just attempting to get e2e working on the 0.3 version I have... (good progress too - will post results method if I get it working)

do you a plan for naming conventions for the path and file names?

angular-seed just uses:  'e2e/*.js'

any thoughts yet?

**Comment by PaulL1** [comment]
In general Josh seems to prefer the files themselves indicate their use - so I'd imagine *.e2e.js.  

Depending on how things go I may end up writing the 0.4 protractor logic, I'd probably name the files *.scenario.js, since that's what I used in #196.  I see that the protractor team tend to use spec.js, but we've used that already for unit tests.  I'm not overly fussed about e2e v's scenario - in my own application I make a distinction between functional tests and e2e tests (i.e. I have some tests that are functional tests but are not e2e), so personally I prefer scenario.
